docs/user-guide/deploying/release-management.md

```diff
@@ -0,0 +1,96 @@
+
+### Using snapshot artifacts
+
+If you want to use changes before they are released, it is also possible to use development snapshot artifacts in your custom projects instead.
+E.g. if the next version will be 1.3.0 use the following versions:
+
+1. Docker images: Update the `openremote/manager` image tag to `develop` in the `docker-compose.yml` file (or environment variable)
+2. Java code: Update the `openremoteVersion` to `1.3.0-SNAPSHOT` in the `gradle.properties` file (requires the Maven repository https://s01.oss.sonatype.org/content/repositories/snapshots to be added to your `project.gradle` file)
+3. TypeScript code: Update the openremote package dependencies using `yarn up -E "@openremote/*@^1.3.0-snapshot"`
+
+### Using ORLib
+
+Optionally you can also add a custom Android or iOS App using the [mobile consoles](../../developer-guide/working-on-the-mobile-consoles) to your custom project.
+For this you can use the ORLib library which simplifies integrating OpenRemote into your App.
+ORLib has its own release cycle because it is maintained in separate repositories and is only released when there are changes.
+
+The Android ORLib is maintained in the [console-android](https://github.com/openremote/console-android) repository and published as a JAR to [Maven Central](https://search.maven.org/artifact/io.openremote/orlib).
+The iOS ORLib is maintained in the [console-ios](https://github.com/openremote/console-ios) repository and published as a CocoaPod to [Trunk](https://cocoapods.org/pods/orlib).
+
+You can find the release notes for ORLib versions on the "Releases" page of the respective GitHub repositories.
+
+### Migration from submodules to versioned artifacts
+
+Previously Git submodules were used in custom projects for versioning.
+This has been replaced with dependencies of versioned artifacts.
+Using submodules for versioning is no longer supported.
+
+If you have an exising custom project using submodules, the easiest way to apply all the changes is to create a new custom project using the [template](https://github.com/openremote/custom-project) and add your customizations to it.
+
+Another way to migrate an existing project is to manually apply all changes of the [commit](https://github.com/openremote/custom-project/commit/...) used for migrating the custom project template.
+
+As you can see in the commit the manual migration steps are:
+
+1. Remove the submodule(s)
+2. Add dependencies to the new artifacts in your build.gradle and package.json files
+3. Add the new "model" and "rest" UI component module code
+4. Update the TypeScript code so classes are loaded from the new modules:
+   1. Add `import rest from "rest";` to your imports if necessary
+   2. Replace `manager.rest.api` with `rest.api` where applicable
+
+If you have additional questions regarding this migration, we encourage you to reach out to our [forum](https://forum.openremote.io/).
+
+## Creating releases
```

Comment:
I have set `mavenLocal()` as last option repository in custom project which could make it difficult to consume local repo artefacts rather than ones from snapshot repo (if same version has been published to both). Given `axion-release-plugin` will always use `-SNAPSHOT` suffix.

Would be nice if we could use `-LOCAL` for publishing to `mavenLocal()` and `-SNAPSHOT` to maven snapshot.